docs/setup/glb-healthcheck-configuration.md

```diff
@@ -0,0 +1,48 @@
+# glb-healthcheck configuration
```

Comment:
Thank you for adding documentation that was missing! Might be useful to link this up from [the current (limited) docs](https://github.com/github/glb-director/blob/master/docs/setup/forwarding-table-config.md#healthcheck-configuration-options), since they are currently pretty sparse and this PR is way more useful 😄

Reply:
Sure